Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -315,6 +319,43 @@ extension ActivityStreamPanel {
         let deleted = profile.prefs.arrayForKey(DefaultSuggestedSitesKey) as? [String] ?? []
         return suggested.filter({deleted.indexOf($0.url) == .None})
     }
+
+    @objc private func longPress(longPressGestureRecognizer: UILongPressGestureRecognizer) {
+        if longPressGestureRecognizer.state == UIGestureRecognizerState.Began {
+            let touchPoint = longPressGestureRecognizer.locationInView(self.view)
+            if let indexPath = tableView.indexPathForRowAtPoint(touchPoint) {
+                if Section(indexPath.section) == .History {
+                    presentContextMenu(history[indexPath.row])
+                }
+            }
+        }
+    }
+
+    private func presentContextMenu(site: Site) {
+        let bookmarkAction = ActionOverlayTableViewAction(title: Strings.BookmarkContextMenuTitle, iconString: "action_bookmark", handler: { action in
+            let shareItem = ShareItem(url: site.url, title: site.title, favicon: site.icon)
+            self.profile.bookmarks.shareItem(shareItem)
+            if #available(iOS 9, *) {
```

Comment:
Now that @thebnich landed the iOS 8 deprecation patch, we don't need to do these checks anymore 😬